src/veh_interact.cpp

```diff
@@ -671,7 +671,7 @@ bool veh_interact::can_install_part() {
     //~ %1$s is quality name, %2$d is quality level
     std::string aid_string = string_format( _( "1 tool with %1$s %2$d" ),
                                             qual.obj().name.c_str(), lvl );
-    std::string str_string = string_format( _( "strength %d" ), str );
+    std::string str_string = string_format( _( "strength ( assisted ) %d" ), str );
```

Comment:
I was assuming the "(assisted)" would only show up if an NPC was around to assist.